scripts/sparkclr-submit.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+function spark_home_error() {
+	echo "[sparkclr-submit.sh] Error - SPARK_HOME environment variable is not export"
+	echo "[sparkclr-submit.sh] Note that SPARK_HOME environment variable should not have trailing /"
+	exit 1
+}
+	
+function java_home_error() {
+	echo "[sparkclr-submit.sh] Error - JAVA_HOME environment variable is not set"
+	echo "[sparkclr-submit.sh] Note that JAVA_HOME environment variable should not have trailing /"
+	exit 1
+}
+	
+function sparkclr_home_error() {
+	echo "[sparkclr-submit.sh] Error - SPARKCLR_HOME environment variable is not set"
+	echo "[sparkclr-submit.sh] SPARKCLR_HOME need to be set to the folder path for csharp-spark*.jar"
+	echo "[sparkclr-submit.sh] Note that SPARKCLR_HOME environment variable should not have trailing /"
+	exit 1
+}
+
+function usage() {
+	echo "Error - usage error."
+	echo "Correct usage is as follows"
+	echo "Example 1:"
+	echo "sparkclr-submit.sh [--verbose] [--master local] [--deploy-mode client] [--name testapp] --exe csdriver.exe sparkclrapp/driver arg1 arg2 arg3"
+	echo "Example 2:"
+	echo "sparkclr-submit.sh [--verbose] [--master local] [--deploy-mode client] [--name testapp] --exe csdriver.exe sparkclrapp/driver.zip arg1 arg2 arg3"
+}
+
+[ "$SPARK_HOME" = "" ] && spark_home_error
+[ "$JAVA_HOME" = "" ] && java_home_error
+[ "$SPARKCLR_HOME" = "" ] && sparkclr_home_error
+
+[ "%SPARK_CONF_DIR%" = "" ] && export SPARK_CONF_DIR="$SPARK_HOME/conf"
+
+. "$SPARK_HOME/bin/load-spark-env.sh"
+
+# Test that an argument was given
+[ $# -le 1 ] && usage
+
+export ASSEMBLY_DIR="$SPARK_HOME/lib"
+
+export SPARK_ASSEMBLY_JAR="0"
+for jar in `ls "$ASSEMBLY_DIR"/spark-assembly*hadoop*.jar`
+do 
+  export SPARK_ASSEMBLY_JAR="$jar"
+done
+if [ "$SPARK_ASSEMBLY_JAR" = "0" ];
+then
+  echo "[sparkclr-submit.sh] Failed to find Spark assembly JAR."
+  exit 1
+fi
+
+export SPARKCLR_JAR=spark-clr-1.4.1-SNAPSHOT.jar
+export SPARKCLR_CLASSPATH="$SPARKCLR_HOME/lib/$SPARKCLR_JAR"
+[ ! "$SPARKCSV_JARS" = "" ] && export SPARKCLR_CLASSPATH="$SPARKCLR_CLASSPATH:$SPARKCSV_JARS"
+export LAUNCH_CLASSPATH="$SPARK_ASSEMBLY_JAR:$SPARKCLR_CLASSPATH"
+
+if [ $1 = "debug" ];
+then
+  "$JAVA_HOME/bin/java" -cp "$LAUNCH_CLASSPATH" org.apache.spark.deploy.csharp.CSharpRunner debug
+else
+
+  # The launcher library prints the arguments to be submitted to spark-submit.sh. So read all the output of the launcher into a variable.
+  export LAUNCHER_OUTPUT=/tmp/spark-class-launcher-output-$RANDOM.txt
+  "$JAVA_HOME/bin/java" -cp "$LAUNCH_CLASSPATH" org.apache.spark.launcher.SparkCLRSubmitArguments "$@" > "$LAUNCHER_OUTPUT"
```

Comment:
It might can be simplified to SPARK_ARGS=`"$JAVA_HOME/bin/java" -cp "$LAUNCH_CLASSPATH" org.apache.spark.launcher.SparkCLRSubmitArguments "$@" > "$LAUNCHER_OUTPUT"`